docs/iris/src/whatsnew/contributions_1.9/newfeature_2015-10-19_newaxis_cube_indexing.txt

```diff
@@ -0,0 +1 @@
+Cube indexing now support newaxis.
```

Comment:
Suggestions:

1. This could do with a leading `* ` to make it a bullet point.
2. "support" -> "supports"
3. "newaxis" -> "`np.newaxis` (aka. `None`)"